spec/game_flow_spec.rb

```diff
@@ -17,6 +17,6 @@
     display = Display.new(output, input)
     game_flow = GameFlow.new(display)
     game_flow.game_flow("tan")
-    expect(output.string).to include("tan")
+    expect(output.string).to include("t a n")
```

Comment:
Yes I did intend that. I hadn't thought that maybe that could be unclear but I suppose it is. I will change it.